regression/cbmc/Anonymous_Struct3/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+CORE broken-smt-backend
```

Comment:
typo in commit message? "easier to repeated build" -> "easier to repeatedly build"? 
"regression test" -> "regression tests"?

Reply:
Thanks a lot!